### PR TITLE
upgrade to angular-15

### PR DESCRIPTION
**Pull request by ct-dixit**

### CODE REVIEW

File: src/app/demo/elements/element-color/element-color.component.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule } from '@angular/router';
+
+@Component({
+  selector: 'app-element-color',
+  standalone: true,
+  imports: [CommonModule, RouterModule],
```
Comment (phoenixcoded20) [inline src/app/demo/elements/element-color/element-color.component.ts:8]: RouterModule needed?

File: src/app/demo/pages/authentication/authentication-routing.module.ts [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+import { RegisterComponent } from './register/register.component';
 
 const routes: Routes = [
   {
     path: '',
     children: [
       {
         path: 'login',
-        loadChildren: () => import('./login/login.module').then((m) => m.LoginModule)
+        component: LoginComponent
```
Comment (phoenixcoded20) [inline src/app/demo/pages/authentication/authentication-routing.module.ts:12]: add back lazy loding. please check other comments

File: src/app/theme/layout/admin/navigation/navigation.ts [hunk 3]
```diff
@@ -88,6 +88,14 @@ const NavigationItems = [
...
         icon: 'ti ti-typography'
       },
+      // {
+      //   id: 'color',
+      //   title: 'Colors',
+      //   type: 'item',
+      //   classes: 'nav-item',
+      //   url: '/color',
+      //   icon: 'ti ti-brush'
+      // },
```
Comment (phoenixcoded20) [inline src/app/theme/layout/admin/navigation/navigation.ts:98]: remove commented code.

**Comment by phoenixcoded20** [comment]
While resolving: @ng-bootstrap/ng-bootstrap@13.1.1
Found: @angular/common@15.0.0

please check if bootstrap has angular 15 compatibility

File: package.json [hunk 4]
```diff
@@ -12,35 +12,35 @@
     "test": "ng test"
```
Comment (phoenixcoded20) [inline package.json:12]: update version to 1.1.0

**Review by phoenixcoded20 — CHANGES_REQUESTED** [review]
PLEASE DO CHANGES

File: src/app/demo/pages/authentication/authentication-routing.module.ts (hunk 2 above)
Comment (ct-dixit) [reply src/app/demo/pages/authentication/authentication-routing.module.ts:12]: done

File: src/app/demo/elements/element-color/element-color.component.ts (hunk 1 above)
Comment (ct-dixit) [reply src/app/demo/elements/element-color/element-color.component.ts:8]: those component in use router link  so this component need router Module

File: src/app/app-routing.module.ts [hunk 5]
```diff
@@ -19,19 +23,19 @@ const routes: Routes = [
       },
       {
         path: 'typography',
-        loadChildren: () => import('./demo/elements/typography/typography.module').then((m) => m.TypographyModule)
+        component: TypographyComponent
```
[the remark this answers is not in the document]
Comment (ct-dixit) [reply src/app/app-routing.module.ts:26]: done

File: src/app/app-routing.module.ts [hunk 6]
```diff
@@ -15,23 +15,23 @@ const routes: Routes = [
...
       },
       {
         path: 'color',
-        loadChildren: () => import('./demo/elements/color/color.module').then((m) => m.ColorModule)
+        loadComponent: () => import('./demo/elements/element-color/element-color.component')
       },
       {
         path: 'tabler',
-        loadChildren: () => import('./demo/elements/tabler/tabler.module').then((m) => m.TablerModule)
+        loadComponent: () => import('./demo/elements/icon-tabler/icon-tabler.component')
```
Comment (phoenixcoded20) [inline src/app/app-routing.module.ts:30]: use external link and remove component

File: src/app/demo/pages/authentication/login/login.component.ts [hunk 7]
```diff
@@ -1,12 +1,12 @@
...
+import { RouterModule } from '@angular/router';
 
 @Component({
   selector: 'app-login',
+  standalone: true,
+  imports: [CommonModule, RouterModule],
   templateUrl: './login.component.html',
   styleUrls: ['./login.component.scss']
 })
-export class LoginComponent implements OnInit {
```
Comment (phoenixcoded20) [inline src/app/demo/pages/authentication/login/login.component.ts:8]: backgroundcolor is different. login and register

**Review by phoenixcoded20 — CHANGES_REQUESTED** [review]

File: src/app/demo/pages/authentication/login/login.component.ts (hunk 7 above)
Comment (ct-dixit) [reply src/app/demo/pages/authentication/login/login.component.ts:8]: done

File: src/app/demo/default/default.component.ts [hunk 8]
```diff
@@ -43,10 +48,12 @@ export type ChartOptions1 = {
```
Comment (phoenixcoded20) [inline src/app/demo/default/default.component.ts:48]: ChartOptions1 is not needed. it can be merge in to ChartOptions and used everywhere.

**Review by phoenixcoded20 — CHANGES_REQUESTED** [review]
- Lets Rename guest and admin component to layout
- ConfigurationComponent - ctror not used.

File: src/app/demo/default/default.component.ts (hunk 8 above)
Comment (ct-dixit) [reply src/app/demo/default/default.component.ts:48]: done

**Pull request merged**